### PR TITLE
some more language translations

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

File: src/main/resources/assets/spookytime/lang/de_de.json [hunk 1]
```diff
@@ -13,6 +14,17 @@
...
+	"item.spookytime.pumpkin_stew": "Kürbissuppe",
+	"item.spookytime.peppermint": "Pfefferminz",
+	"item.spookytime.wooden_club": "Holzschläger",
+	"item.spookytime.stone_club": "Steinschläger",
+	"item.spookytime.iron_club": "Eisenschläger",
+	"item.spookytime.gold_club": "Goldschläger",
+	"item.spookytime.reapers_scythe": "Sense des Sensenmannes",
+	"item.spookytime.spooky_trumpet": "Gespenstische Trompete",
+	"item.spookytime.witch_water_bucket": "Hexewassereimer",
+	"item.spookytime.blood_bucket": "Bluteimer",
```
Comment (valoeghese) [inline src/main/resources/assets/spookytime/lang/de_de.json:26]: This translation feels off but idk

**Review by emilyploszaj — APPROVED** [review]
All looks good.

File: src/main/resources/assets/spookytime/lang/de_de.json (hunk 1 above)
Comment (Shnupbups) [reply src/main/resources/assets/spookytime/lang/de_de.json:26]: Would 'Schnitter-Sense' or 'Sense des Schnitter' be a better translation? Or perhaps 'Sense des Todes'?

Comment (valoeghese) [reply src/main/resources/assets/spookytime/lang/de_de.json:26]: Yeah but if I use this translation [for the scythe one] it's hilarious

Comment (valoeghese) [reply src/main/resources/assets/spookytime/lang/de_de.json:26]: and I was talking about the blood bucket translation not the scythe one

Comment (Shnupbups) [reply src/main/resources/assets/spookytime/lang/de_de.json:26]: Perhaps 'Eimer mit Blut'?

**Review by Juuxel — APPROVED** [review]

File: src/main/resources/assets/spookytime/lang/de_de.json (hunk 1 above)
Comment (valoeghese) [reply src/main/resources/assets/spookytime/lang/de_de.json:26]: I'll keep it like this for now. If any Germans complain we can change it.

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**